push_feedback_to_all.sh

```diff
@@ -5,14 +5,18 @@
 feedback_filename=$1
 
```

Comment:
can you add a comment at the top to indicate where this should be run from?

Reply:
It has the same usage as before, but I will cover this on readme file